SU-SWS/stanford_opportunity
# SOE-2372: Doc and version updates for 7.x-1.0-alpha1

# READY FOR REVIEW

# Summary
- Release template and associated version updates for push to production.

# Needed By (11/13)
- I hope to stage a 7.x-1.0-alpha1 release on jse-soe-dev by Monday, so you can commit it to the gitolite repo.

# Urgency
- Between 1 and 10, perhaps a 7.

# Steps to Test

1. Check out this branch
2. Manually review the diff for anything, checking for anything that was *not* documentation or version related
3. Review the diff with `--name-only` to verify that only `CHANGELOG`, `README`, and `.info` files were modified
4. Recursively grep for `7.x-1.0-dev` to ensure that it was only found in `CHANGELOG`

Also be sure the Release Template below looks accurate:

### What (Changelog)
- Remove entries that have been imported, but are no longer found in the feed source, solo.stanford.edu.

### So what
- This work ensures that students looking for off-campus opportunities on the SOE home site do not find opportunities that have since closed.

### Now what
- If this module has already been enabled on a site, you'll need to visit admin/structure/feeds/stanford_opportunity_helper/settings/FeedsNodeProcessor and manually 
select "Delete non-existent nodes" if not already selected, then click Save.  You'll also need to revert the stanford_opportunity_importer feature.

### New Requirements and Risks (Changelog)
- This update requires a manual change for sites already using an existing opportunities importer.

### Diff
- First release.

# Affected Projects or Products
- SOE home site

# Associated Issues and/or People
- https://stanfordits.atlassian.net/browse/SOE-2372
- https://stanfordits.atlassian.net/browse/SOE-2310

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
